views/partials/dashboardResults.ejs

```diff
@@ -1,103 +1,158 @@
+                    href="/dashboard?searchText=<%= searchCriteria %>&sortOrder=<% if (sortOrder == 3) { %>-<%}%>3">
+                    Service <span class="visually-hidden">
+                        <% if (sortOrder==3) { %> ascending<%}%>
+                                <% if (sortOrder==-3) { %> descending<%}%>
+                    </span>
+                </a></th>
+            <th>Status</th>
+        </tr>
     </thead>
     <tbody>
-    <% for (i=0;i<users_applications.length;i++) { %>
-    <tr>
-        <td class="appDate">
-                        <span class="hide-col-sm hide-col-md">
-                            <%= dayjs(users_applications[i].createdDate).format('DD-MM-YY') %>
-                        </span>
-                        <span class="hide-col-lg">
-                            <span class="appday"><%= dayjs(users_applications[i].createdDate).format('DD/MM') %></span>
-
-                            <span class="appyear"><%= dayjs(users_applications[i].createdDate).format('YYYY') %></span>
-                        </span>
-        </td>
-      <td class="appRef"><a class="govuk-link"
-          href="/open-application/<%= users_applications[i].unique_app_id %>"> <%= users_applications[i].unique_app_id %> </a>
-      </td>
-      <td class="appService hide-col-sm hide-col-md"><%= users_applications[i].applicationtype %></td>
-      <td class="appDocCount hide-col-sm hide-col-md"><%= users_applications[i].doc_count %></td>
-      <td class="appPayment">£<%= parseFloat(users_applications[i].payment_amount + 0).toFixed(2) %></td>
-      <td class="appService hide-col-sm hide-col-md"><%= users_applications[i].user_ref || 'Not available' %></td>
-      <td><%= users_applications[i].app_status.text || 'Not available' %></td>
-      <% if (users_applications[i].tracking_ref && users_applications[i].main_postcode) { %>
-      <td class="trackingRef"><a href="https://www.dxdelivery.com/?tn=<%=users_applications[i].tracking_ref%>&tpc=<%=users_applications[i].main_postcode%>"
-                                 target="_blank"><%= users_applications[i].tracking_ref %> </a></td>
-      <% }else if (users_applications[i].tracking_ref) { %>
-      <td class="trackingRef"><a href="https://www.dxdelivery.com/consumer/my-tracking/"
-                                 target="_blank"><%= users_applications[i].tracking_ref %> </a></td>
-      <% }else{ %>
-      <td class="trackingRef"> Not available</td>
-      <% } %>
-    </tr>
-    <% } %>
+        <% for (i=0;i<users_applications.length;i++) { %>
+            <tr style="vertical-align: text-top">
+                <td class="appDate">
+                    <%= dayjs(users_applications[i].createdDate).format('DD-MM-YYYY') %>
+                </td>
+                <td>
+                    <% if (users_applications[i].applicationtype === 'e-Apostille') { %>
```

Comment:
You could probably replace this bloaty if statement with a getUrlPath function on the users_application object.  The  if statement could then be contained within there allowing you to replace this block with something like (or equivalent syntax for inline variables)

`....
href="<%= users.applications[i].getUrlPath/>">
....`

Reply:
Resolved with the inclusion of the viewAppUrl property